client/lib/wp/sync-handler/utils.js

```diff
@@ -1,11 +1,10 @@
 /**
  * External dependencies
  */
-import sha1 from 'js-sha1';
+import sha1 from 'hash.js/lib/hash/sha/1';
```

Comment:
>in fact, we could get rid of sync-handler as well. In this way, we could clean this module completely.

Yeah, I figured sync handler is on its way out. Most of the pieces are already there with the newer custom post type screen (i.e. caching post responses for future use). I think the remaining issue is whether we need to preserve existing UIs (especially with newer features like re-Publicize) and how we extend CPT components to support those cases. Regardless, it's a bigger issue than one I want to address here, and I think it'll be good to keep the sync handler around until a full replacement can be made.